src/util/hass-media-player-model.js

```diff
@@ -178,13 +178,19 @@ export default class MediaPlayerEntity {
   }
 
   togglePower() {
-    if (this.isOff) {
+    if (this.supportsTurnOn && this.supportsTurnOff) {
```

Comment:
This condition may not be necessary. The Plex integration seems to be the only media_player that supports 'turnoff' but not 'turnon'. Used >grep -rl 'SUPPORT_TURN_ON' homeassistant/components/*/media_player.py

Reply:
Honestly the Plex integration should probably not advertise as supporting either power on/off as it doesn't have the ability to do either.

Reply:
There are more that can't turn on. Philips js, arcam. They can support it with external wol or it blasters.